### PR TITLE
Accessibility Guidelines

### PR DESCRIPTION
**Pull request by story645**
Images put out by the official mpl accounts should be accessible when possible. Adding this here 'cause forgot to mention it to Peter.

### CODE REVIEW

File: communications_guidelines.md [hunk 1]
```diff
@@ -34,7 +34,16 @@ These guidelines are applicable when acting as a representative of Matplotlib (f
...
         - https://help.twitter.com/en/rules-and-policies/twitter-rules
         - https://help.instagram.com/477434105621119
-        
+
+## Accessibility
+- add alt text to images and videos when the platform allows
+    - https://webaim.org/techniques/alttext/
+- warn on bright, strobing, images & turn off autoplay if possible
+- for images made by the social media team:
+  - make graphic percievable to person who cannot perceive color well, due to color-blindness or low vision
```
Comment (QuLogic) [inline communications_guidelines.md:43]:
```suggestion
  - make graphic perceivable to people who cannot perceive color well, due to color-blindness or low vision
```

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
I'm unclear what the merge rules are here. Is this a one-approval merge like the docs?

**Comment by jklymak** [comment]
No I think it's a majority of Tom, Eric, Ryan and you.

**Comment by story645** [comment]
there maybe should be a separate discussion of whether these go here or should be moved to the brochure site...it's here 'cause these are process docs, but there are loads of PR process docs in the main site too.

**Review by dopplershift — APPROVED** [review]

**Comment by jklymak** [comment]
@tacaswell @efiring, This OK w/ you both?

**Comment by efiring** [comment]
Looks fine to me.

**Comment by tacaswell** [comment]
I am not sure that this should be in the governance repo rather than our normal documentation, but will move this whole page out at a later step.

**Pull request merged**